core/libp2p/libp2p.hpp

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_LIBP2P_HPP
+#define KAGOME_LIBP2P_HPP
+
+#include <memory>
+
+#include <rxcpp/rx.hpp>
```

Comment:
include only observable

Reply:
I get a compile error when trying to include only it, and searches for the headers, which are needed to get rid of it, were a failure